creator/tasks.py

```diff
@@ -291,8 +291,7 @@ def slack_notify_task():
     logger.info("Running the slack_notify job")
 
     try:
-        # Call daily notification function here
-        pass
+        summary_post(Study.objects.all())
```

Comment:
Can the `Study.objects.all()` be moved into the `summary_post` function? That way the database query will run inside the task instead of inside the scheduler.